spec/models/user_spec.rb

```diff
@@ -0,0 +1,134 @@
+require 'rails_helper'
+
+describe User do
+  let(:user){build(:user)}
+
+  it "should be valid with default attributes" do
```

Comment:
I really like your thorough testing for the bounds of your user attribute validations.